deep-learning/_papers/2017-02-28-single-shot-detector-ssd.md

```diff
@@ -0,0 +1,47 @@
+
+
+```
+---
+layout: review
+title:  SSD: Single Shot MultiBox Detector (2016)
```

Comment:
Surround the title and your name with quotes (since there are special characters in them)